first-analysis-steps/lhcb-dev.md

```diff
@@ -175,8 +185,10 @@ package and version:
 ```
 
 {% callout "What if `git` asks for my password?" %}
+
 Make sure you succesfully completed the instructions under [initial
 setup](#initial-setup).
```

Comment:
This one doesn't seem to work either

Reply:
None of the new panels were being given ID of the heading which could be referenced. I've fixed it.